docs/plugin-transform-typescript.md

```diff
@@ -79,8 +83,75 @@ Replace the function used when compiling JSX expressions.
 
 This is so that we know that the import is not a type import, and should not be removed
 
+## TypeScript Compiler Options
+
+The official TypeScript compiler has many [options][tsc-options] which allow configuring both
+the compiler and the type checker. While many of them don't make sense for Babel (for example,
+`--allowUnusedLabels`), some of them might be useful. Their behavior can be achieved in Babel
+using different configuration options or plugins.
+
+- `--alwaysStrict`
+  You can use the `strictMode` [parser option](https://babeljs.io/docs/en/babel-parser#options):
+  ```js
+  module.exports = {
+    parserOpts: { strictMode: true },
+  };
+  ```
+- `--downlevelIteration`
+  You can use the `@babel/plugin-transform-for-of` plugin. If you are using `@babel/preset-env`, `for...of` is already transpiled using iterators when it isn't supported by your compilation target.
+- `--emitDecoratorMetadata`
+  This option isn't supported by an official Babel package since it is not part of the decorators proposal, but it is a TypeScript-specific addition.
+  If you rely on this feature, you can use the community plugin [babel-plugin-transform-typescript-metadata](https://github.com/leonardfactory/babel-plugin-transform-typescript-metadata#readme).
```

Comment:
@leonardfactory Can we mention your plugin?

Reply:
@nicolo-ribaudo Absolutely! Thank you for asking 👍

Reply:
Does this plugin work the same way with babel standalone?